src/main/java/com/danielagapov/spawn/DTOs/ChatMessageLikesDTO.java

```diff
@@ -0,0 +1,7 @@
+package com.danielagapov.spawn.DTOs;
+
+import java.io.Serializable;
+import java.util.UUID;
+
+public record ChatMessageLikesDTO (
```

Comment:
Let's have this extend the entity, `ChatMessageLikes`. Then, if the DTO needs specific data fields, we can add them. This is how we can solve the issue of having this empty DTO, which can lead to client-side confusion of getting nothing back.